apps/client/src/components/Nav.test.js

```diff
@@ -24,30 +25,36 @@ const Wrapper = ({ children }) => {
 };
 
 it("Nav should render with no props", () => {
-  render(
+  const { queryByTestId } = render(
     <Wrapper>
       <Nav />
     </Wrapper>
   );
 
-  expect(screen.findByText("Vorige")).not.toBeFalsy();
-  expect(screen.findByText("Volgende")).not.toBeFalsy();
+  const prevButton = queryByTestId(PREV_BUTTON);
+  const nextButton = queryByTestId(NEXT_BUTTON);
+
+  expect(nextButton).not.toBeInTheDocument();
```

Comment:
if nextButton is only used once I would consider placing it in expect fn directly.
`expect(queryByTestId(NEXT_BUTTON)).not.toBeInTheDocument();`
Minor detail though.

Reply:
Great catch! Fixed